source/credentials_provider_sts_web_identity.c

```diff
@@ -934,6 +934,20 @@ static int s_generate_uuid_to_buf(struct aws_allocator *allocator, struct aws_by
     return AWS_OP_SUCCESS;
 }
 
+static void s_check_or_get_with_env(
```

Comment:
trivial: why not just return `aws_string *` instead of passing the `aws_string **out`?

Reply:
Thanks, updated.